Solutions/Votiro/Analytic Rules/VotiroFileBlockedInEmail.yaml

```diff
@@ -0,0 +1,41 @@
+id: 0b8b91de-c63e-4bc2-b5f4-b15d3b379ec9
+name: Votiro - File Blocked in Email
+description: |
+ 'The analytic rule is designed to identify when an email is blocked by Votiro Sanitization Engine policy. The rule generates an alert when an email is blocked after Sanitization process which is not password protected and has a from field meaning its a valid email. More information in terms of details about the blocked attachment, such as the file name and hash, and information about the email, such as the sender and recipient(s). The alert also includes information about the Votiro policy that blocked the attachment and provides a link to additional details about the incident.'
+severity: Low
+requiredDataConnectors:
+  - connectorId: Votiro
+    dataTypes:
+     - CommonSecurityLog 
+queryFrequency: 10m
+queryPeriod: 10m
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - CommandAndControl
+  - DefenseEvasion
+  - Impact
+  - InitialAccess
+relevantTechniques:
+  - T0885
+  - T1036
+  - T1027
+  - T1486
+  - T1566
+query: let Votiro_view  = view () {
+  VotiroEvents
+   | parse-where SanitizationResult with "Blocked" blocked
```

Comment:
hi @devikamehra,
We cannot use the "has" function with the "parse-where" operator.
The "parse-where" operator is used to extract fields from the raw data using regular expressions and apply filtering conditions to those fields. The output of the "parse-where" operator is a set of fields that are derived from the raw data.
On the other hand, the "has" function is used to check if a field contains a particular value. It operates on a field that already exists in the data and cannot be used to filter fields that are derived from the raw data using the "parse-where" operator.
If we want to use 'has' function we have to use 'where' instead of 'parse-where'.
Thanks,
Sambhrant.